src/main/java/org/kitteh/irc/client/library/implementation/Config.java

```diff
@@ -135,7 +135,7 @@ public String toString() {
     static final Entry<StringConsumerWrapper> LISTENER_OUTPUT = new Entry<>(null, StringConsumerWrapper.class);
     static final Entry<Integer> MESSAGE_DELAY = new Entry<>(Client.DEFAULT_MESSAGE_DELAY, Integer.class);
     static final Entry<String> NICK = new Entry<>("Kitteh", String.class);
-    static final Entry<String> REAL_NAME = new Entry<>("Kitteh", String.class);
+    static final Entry<String> REAL_NAME = new Entry<>("Kitteh v" + KICLProperties.DEFAULT_VERSION, String.class);
```

Comment:
How about just Kitteh 2.4.0 instead of v2.4.0? Maybe also inclusion of kitteh.org url? :)